docs/pages/versions/unversioned/sdk/notifications.mdx

```diff
@@ -19,7 +19,7 @@ import { Collapsible } from '~/ui/components/Collapsible';
 
 The **`expo-notifications`** provides an API to fetch push notification tokens and to present, schedule, receive and respond to notifications.
 
-> Migrating from Expo's `LegacyNotifications` module? [Here's a guide to help make the transition as easy as possible](https://github.com/expo/fyi/blob/master/LegacyNotifications-to-ExpoNotifications.mdx).
+> Migrating from Expo's `LegacyNotifications` module? [Here's a guide to help make the transition as easy as possible](https://github.com/expo/fyi/blob/master/LegacyNotifications-to-ExpoNotifications.md).
```

Comment:
```suggestion
> Migrating from Expo's `LegacyNotifications` module? [Here's a guide to help make the transition as easy as possible](https://expo.fyi/legacy-notifications-to-expo-notifications).
```

BTW. Maybe, in general we should use the short form to avoid the extension problem in general?

Reply:
Yep, that is great!

I didn't pay much attention to the short form links and yeah with it, you are right, the extension problem is gone. I'll update rest of the docs. Thanks for the suggestion 🙏